docs/targets.rst

```diff
@@ -53,3 +53,33 @@ by passing them to Actions::
         Enter.the_secret("reallySecurePassword!!").into_the(PASSWORD_FIELD),
         Click.on_the(SIGN_IN_BUTTON),
     )
+
+The resulting log::
+
+    Webster enters "webster_1987" into the username field.
+    Webster enters "[CENSORED]" into the password field.
+    Websert clicks on the "Sign In" button.
+
+By default the :ref:`target` will use the locator string as a human-readable
+target name in the absense of providing one. This can be convenient if your
```

Comment:
Two small nitpicks on this bit—i've been trying to be consistent to capitalize ScreenPy-related concepts like "Target" and "Actions" etc.; can you capitalize "target" here?

The other is a spelling error—"absense" should be "absence". Though i do have a sense of abs.

Reply:
updated